doc/1.15/samtools-mpileup.html

```diff
@@ -374,7 +374,7 @@ <h1 id="OPTIONS"><a href="#OPTIONS">OPTIONS</a></h1>
 BAQ is calculated using the probabilistic realignment method described
 in the paper &ldquo;Improving SNP discovery by base alignment quality&rdquo;,
 Heng Li, Bioinformatics, Volume 27, Issue 8
-&lt;<a href="https://doi.org/10">https://doi.org/10</a>.1093/bioinformatics/btr076&gt;
+&lt;<a href="https://doi.org/10">https://doi.org/10.1093/bioinformatics/btr076</a>&gt;
```

Comment:
Probably the `href="…"` needs adjusting too (x 2).

Reply:
Oops, yes probably a good idea.  It should be fixed now.

Reply:
:smile:

See also jmarshall/manconvert@3ed7a1ba6f891bcdd1c3e4c191de79c58628b38c.